src/main/java/roomescape/service/ReservationService.java

```diff
@@ -28,10 +38,6 @@ public ReservationResponse reserve(ReservationCreateRequest request) {
         return new ReservationResponse(id, name, reservationDate, reservationTime);
     }
 
-    public List<Reservation> showReservations() {
-        return Collections.unmodifiableList(reservations);
-    }
-
     public void cancelReservation(Long reservationId) {
         Reservation reservation = reservations.stream()
                 .filter(r -> r.getId().equals(reservationId))
```

Comment:
[의견 🗣️ ]

해당 로직을 조금 더 객체지향적으로 바꾸어볼 수 있을 것 같아요. id를 꺼내오는 게 아니라 예약에게 메시지를 던져 물어보는 방식은 어떨까요?

```suggestion
                .filter(r -> r.isSame(reservationId))
```

Reply:
아.. 부끄럽네요. 프리코스를 겪으면서 객체지향적으로 설계하자 다짐하고 생각했는데...
귀찮았던 점이 컸던 것 같습니다. 바꿔보겠습니다!!

Reply:
아무래도 웹 어플리케이션을 설계하고, 각 계층에 신경쓰다보면 객체를 능동적 행위자가 아니라 데이터로 보게되는 경우가 종종 있죠 ㅎㅎ 저도 개선하려 하는 습관이지만 쉽지 않은 것 같아요! 파도가 리뷰의 포인트를 바로 캐치한 것만으로 충분히 잘하고 계시다고 생각합니다.